pymchelper/readers/shieldhit.py

```diff
@@ -9,9 +9,371 @@
+            detector.data *= np.float64(1.60217662e-10)
+            detector.units[0:4] = SHBinaryReader.get_estimator_units(detector.geotyp)
+            detector.units[4:6] = SHBinaryReader.get_detector_unit(detector.dettyp, detector.geotyp)
+            detector.title = detector.units[5]
+
+
+class SHBDOTagID(IntEnum):
+    """ List of Tag ID numbers. Must be synchronized with sh_detect.h in SH12A.
+    """
+
+    # Hex values are used for better regognition in binary files, should they be inspected by humans.
+    # Group 0x0000 - 0x00FF : Miscellaneous info
+    shversion = 0x00    # [char*] full version string of shield-hit12a
+    shbuilddate = 0x01  # [char*] date of build
+    filedate = 0x02     # [char*] bdo file creation date, RFC 2822 compliant
+    user = 0x03         # [char *] optional login name
+    host = 0x04         # [char *] optional host where this file was created
+
+    # Group 0xCC00 - 0xCCFF : Configuration
+    dele = 0xCC00
+    demin = 0xCC01
+    itypst = 0xCC02
+    itypms = 0xCC03
+    oln = 0xCC04
+    inucre = 0xCC05
+    iemtrans = 0xCC06
+    iextspec = 0xCC07
+    intrfast = 0xCC08
+    intrslow = 0xCC09
+    apzlscl = 0xCC0A
+    ioffset = 0xCC0B
+    irifimc = 0xCC0C
+    irifitrans = 0xCC0D
+    irifizone = 0xCC0E
+    ext_nproj = 0xCC0F
+    ext_ptvdose = 0xCC10
+    ixfirs = 0xCC11
+
+    # Group 0xCE00 - 0xCEFF : CT specific tags
+    ct_ang = 0xCE00   # holds two doubles with the couch and gantry angle
+    ct_icnt = 0xCE01  # holds three
+    ct_len = 0xCE02   # holds three
+
+    # Group 0xEE00 - 0xEEFF : Estimator specific tags
+    # estimator specific tags
+    est_geotyp = 0xEE00    # may differ from det_geotyp in case of spc
+    est_pages = 0xEE01     # number of detectors / pages for this estimator
+
+    # Group 0xDD00 - 0xDDFF : Detector/page specific tags
+    det_geotyp = 0xDD00      # may differ from est_geotyp in case of spc
+    det_nbin = 0xDD01        # idet(1-3) (len=3) number of bins x,y,z
+    det_part = 0xDD02        # idet(4) particle type which was scored
+    det_dtype = 0xDD03       # idet(5) detector type
+    det_partz = 0xDD04       # idet(6)
+    det_parta = 0xDD05       # idet(7)
+    det_dmat = 0xDD06        # idet(8)
+    det_nbine = 0xDD07       # idet(9)
+    det_difftype = 0xDD08    # idet(10)
+    det_zonestart = 0xDD09   # idet(11)
+    det_dsize = 0xDD0A       # idet(12)
+    det_dsizexyz = 0xDD0B    # idet(13)
+
+    det_xyz_start = 0xDD0C   # det(1-3)
+    det_xyz_stop = 0xDD0D    # det(4-6)
+    det_dif_start = 0xDD0E   # det(7)
+    det_dif_stop = 0xDD0F    # det(8)
+    det_voxvol = 0xDD10      # det(9)
+
+    det_data = 0xDDBB        # data block
+
+    # Group 0xAA00 - 0xAAFF : Runtime variables
+    rt_nstat = 0xAA00        # number of actually simulated particles
+    rt_time = 0xAA01         # [usignend long int] optional runtime in seconds
+
+
+mapping = {SHBDOTagID.shversion: "mc_code_version",
```

Comment:
This mapping seems to be not used. Can it be commented out for future reference ?